analyses/sample-distribution-analysis/results/primary_sites_counts.tsv

```diff
@@ -1,21 +1,21 @@
 primary_site	number_of_types	max_type	second_max_type
```

Comment:
This seems broken?

Reply:
Yep definitely broken https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/214